hubblestack/files/hubblestack_nova/misc.py

```diff
@@ -279,12 +279,22 @@ def world_writable_file(reason=''):
     return True if result == '' else result
 
 
-def system_account_non_login(reason=''):
+def system_account_non_login(non_login_shell='/sbin/nologin', max_system_uid='500', except_for_users=''):
```

Comment:
The value of non_login shell is '/sbin/nologin' in other operating systems. But in ubuntu16, the path of non login shell is '/usr/sbin/nologin'.
So, I now we are taking the path of non_login_shell as an argument. It's default value is still '/sbin/nologin', so it won't affect existing profiles.